### DISCUSSION TITLE
Multi map change issue

### DISCUSSION THREAD

**Discussion by alarik84** [humbertogontijo/homeassistant-roborock#332] [Q&A] [answered]
Hi.
First of all thank you for your great job creating this integration. It should be officialy supported by the manufacturer.

My question is: how to configure switching mulitple maps in home assistant. I have configured it following some disucssions but I can;'t figure out how to do it correctly. My map switch configuration doesn't work, when I use button to change the floor it doesn't work unless I reload the configuration. My code is standard:

`map_source:
  camera: camera.roborock_s7_map
calibration_source:
  camera: true
append_icons: true
icons:
  - icon: mdi:numeric-0-box
    tooltip: Map 0
    tap_action:
      action: call-service
      service: vacuum.send_command
      service_data:
        entity_id: vacuum.roborock_s7
        command: load_multi_map
        params: 0
  - icon: mdi:numeric-1-box
    tooltip: Map 1
    tap_action:
      action: call-service
      service: vacuum.send_command
      service_data:
        entity_id: vacuum.roborock_s7
        command: load_multi_map
        params: 1`

I'd appreciate any hints.

**Comment by humbertogontijo** [comment, marked as answer]
Use service roborock.vacuum_load_multi_map instead

**Reply by alarik84** [reply]
Thank you very much. 
It required some more twaks, but now I understand the principal.
